subsys/random/Kconfig

```diff
@@ -88,6 +88,7 @@ config CSPRNG_ENABLED
 choice CSPRNG_GENERATOR_CHOICE
 	prompt "Cryptographically secure random generator"
 	default HARDWARE_DEVICE_CS_GENERATOR
+	default TEST_CSPRNG_GENERATOR
```

Comment:
Curious to know: does this actually default to `TEST_CSPRNG_GENERATOR` if `!ENTROPY_HAS_DRIVER` even though the conditions of `CTR_DRBG_CSPRNG_GENERATOR` (which is placed before `TEST_CSPRNG_GENERATOR`) are satisfied?

Reply:
The ordering of options within a `choice` has no impact on the value chosen, that is determined by the ordering of the `default` statements.

Reply:
It does have an impact on the value chosen if there is no `default`. Now it felt like an in-between case, which is why I got wondering about the behavior it produces.